sentry.client.config.js

```diff
@@ -5,7 +5,7 @@
 import * as Sentry from '@sentry/nextjs'
 
 Sentry.init({
-  dsn: process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN,
+  dsn: 'https://78ce09ae2f694f0e8981cf5cec6fc2c7@o1008683.ingest.sentry.io/4504810271408128',
```

Comment:
using the env value in the sentry config files didn't seem to be working. I discussed whether there may be any potential ramifications of hard coding this value with hans. he concluded that the risks are minimal.

https://docs.sentry.io/product/sentry-basics/concepts/dsn-explainer/#dsn-utilization:~:text=DSNs%20are%20safe,DSN